Source/WebKit/UIProcess/WebPageProxy.cpp

```diff
@@ -3977,10 +3977,9 @@ void WebPageProxy::sendWheelEvent(WebCore::FrameIdentifier frameID, const WebWhe
     internals().wheelEventActivityHysteresis.impulse();
 #endif
 
-    Ref connection = m_legacyMainFrameProcess->connection();
     if (drawingArea()->shouldSendWheelEventsToEventDispatcher()) {
         sendWheelEventScrollingAccelerationCurveIfNecessary(event);
-        connection->send(Messages::EventDispatcher::WheelEvent(webPageIDInMainFrameProcess(), event, rubberBandableEdges), 0, { }, Thread::QOS::UserInteractive);
```

Comment:
One thing missing that I could not find in the PageProxy/ProcessProxy `send()` API was the ability to set the `Thread::QOS`, to align with the original code that sends through the connection directly.

Maybe add something like a `WebPageProxy::sendWithThreadQOS(...)`? Storing the QOS value in the `AuxiliaryProcessProxy::PendingMessage` struct before passing to the connection?